components/Dialogs/src/MDCDialogPresentationController.h

```diff
@@ -95,6 +95,13 @@
  */
 @property(nonatomic, assign) MDCShadowElevation dialogElevation;
 
+/**
+ Customize the shadow color of dialog.
```

Comment:
Tip: Nit: Consider copy editing to match the better docs on the AlertController API.

Reply:
Done